<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPage_Flac</class>
 <widget class="QWidget" name="flacConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLAC encoding configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="flacCompressionLabel">
       <property name="text">
        <string>Compression:</string>
       </property>
       <property name="buddy">
        <cstring>flacCompressionSlider</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="flacCompressionSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="flacCompressionSpin">
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="flacGainGroup">
     <property name="title">
      <string>ReplayGain</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QGridLayout" name="formLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="flacGainLabel">
          <property name="text">
           <string>Calculate gain:</string>
          </property>
          <property name="buddy">
           <cstring>flacGainCbx</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="flacGainCbx">
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>flacCompressionSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>flacCompressionSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flacCompressionSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>flacCompressionSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
